hbase-server/src/main/java/org/apache/hadoop/hbase/mob/MobFileCompactionChore.java

```diff
@@ -84,10 +84,10 @@ public MobFileCompactionChore(Configuration conf, int batchSize) {
   protected void chore() {
 
     boolean reported = false;
-
-    try (Connection conn = master.getConnection();
-        Admin admin = conn.getAdmin();) {
-
+    Admin admin = null;
+    try {
```

Comment:
Can we just use try with resources on the `Admin` object? Do we actually use the Connection at all?

Reply:
Done